backend/routes/graph.py

```
@@ -12,12 +13,21 @@
 
 @router.get("/{user_id}")
 def get_user_graph(user_id: str):
-    return get_graph(user_id)
+    try:
+        return get_graph(user_id)
+    except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)